c7n/resources/rds.py

```diff
@@ -46,6 +46,7 @@
 import logging
 import re
 
+from pkg_resources import parse_version
```

Comment:
ic, for this use case loose version is appropriate and fine for the db engine version comparisons, always good to look over implementations when comparing

https://github.com/python/cpython/blob/master/Lib/distutils/version.py

the pkg_resources ends up deferring mostly to
https://github.com/pypa/setuptools/blob/master/pkg_resources/_vendor/packaging/version.py

also fwiw, strict does allow alpha chars but requires trailing num. wrt to comparison the primary delta isn't accuracy but strictness.